instrumentation/elasticsearch/elasticsearch-rest-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/rest/v5_0/Elasticsearch5RestClientInstrumentationModule.java

```diff
@@ -5,6 +5,7 @@
 
 package io.opentelemetry.javaagent.instrumentation.elasticsearch.rest.v5_0;
 
+import static io.opentelemetry.javaagent.instrumentation.api.Java8BytecodeBridge.currentContext;
```

Comment:
In some places above you use `Context.current()` directly. What are the rules? How to choose when to use which? Are they written in CONTRIBUTING.md?

Reply:
See the Javadoc for `Java8BytecodeBridge`. This situation is not good though. #1840 looks promising, less code in advice, less (no?) need for `Java8BytecodeBridge`.

Reply:
If developers know about `Java8BytecodeBridge` they can read javadoc and understand what is going on. But how can they find it in the first place? :)

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/contributing/writing-instrumentation.md#java-agent-instrumentation-gotchas